lighthouse/classes/events/biosero/source_completed.py

```diff
@@ -1,29 +1,51 @@
 import logging
-from typing import Dict
+from typing import Dict, Any
 
 from lighthouse.classes.plate_event import PlateEvent
-from lighthouse.messages.message import Message
+
+from lighthouse.classes.event_properties.definitions import (  # type: ignore
+    PickedSamplesFromSource,
+    RobotUUID,
+    RunInfo,
+    SourcePlateUUID,
+    UserID,
+    PlateBarcode,
+    RunID,
+    RobotSerialNumber,
+)
 
 logger = logging.getLogger(__name__)
 
 
 class SourceCompleted(PlateEvent):
-    def __init__(self, name: str) -> None:
-        super().__init__(name=name, plate_type=PlateEvent.PlateTypeEnum.SOURCE)
+    def __init__(self, event_type: str) -> None:
+        super().__init__(event_type=event_type, plate_type=PlateEvent.PlateTypeEnum.SOURCE)
```

Comment:
Do you mean overriding the value? Because I think is defining them for the first time. This is calling the parent method of ```__init__``` because we are redefining``` __init__``` here.